frontend/src/views/Sponsor.vue

```diff
@@ -0,0 +1,204 @@
+<template>
+  <div>
+    <header id="showcase">
+      <v-img max-width="80vw" max-height="30vh" contain src="@/assets/csesocwhiteblue.png" />
+    </header>
+    <v-container class="margin" fluid>
+      <h1 class="border text-h1 font-weight-bold" style="padding:25px;">Sponsors</h1>
+      <h2 class="text-h4">Principal <br> Sponsors</h2>
+      <v-container class="border" fluid>
+          <div v-for="(sponsor, index) in tierOne" :key="sponsor.id" :style="marginStyle(index, largeLogoFilter)">
+              <div class="logo-margin">
+                <img class="large-logo" :src="sponsor.logo" v-on:click="onClickModal(sponsor)"/>
```

Comment:
Make sure you're loading the base64 images the way it's being done in the dev branch. Currently they aren't loading.

Reply:
Fixed!